draft-wussler-openpgp-pqc.md

```diff
@@ -1281,114 +1197,105 @@ consists of:
    big-endian encoded integer, whose length depends on the algorithm used as
    specified in {{tab-ecdsa-artifacts}}.
 
- - A fixed-length octet string containing the Dilithium secret key, whose
-   length depends on the algorithm ID as specified in
-   {{tab-dilithium-artifacts}}.
+ - A fixed-length octet string containing the ML-DSA secret key, whose length
+   depends on the algorithm ID as specified in {{tab-mldsa-artifacts}}.
+
+# SLH-DSA
 
-# SPHINCS+
+## The SLH-DSA Algorithms {#slhdsa}
 
-## The SPHINCS+ Algorithms {#algo-sphincs}
+The following table describes the SLH-DSA parameters and artifact lengths:
 
-The following table describes the SPHINCS+ parameters and artifact lengths:
+{: title="SLH-DSA parameters and artifact lengths in octets. The values equally apply to the parameter IDs of SLH-DSA-SHA2 and SLH-DSA-SHAKE." #slhdsa-artifact-lengths}
+Parameter ID reference | Parameter name suffix | SLH-DSA public key | SLH-DSA secret key | SLH-DSA signature
+----------------------:| ---------------------:| ------------------ | ------------------ | ------------------
+1                      | 128s                  | 32                 | 64                 | 7856
+2                      | 128f                  | 32                 | 64                 | 17088
+3                      | 192s                  | 48                 | 96                 | 16224
+4                      | 192f                  | 48                 | 96                 | 35664
+5                      | 256s                  | 64                 | 128                | 29792
+6                      | 256f                  | 64                 | 128                | 49856
 
-{: title="SPHINCS+ parameters and artifact lengths in octets. The values equally apply to the parameter IDs of SPHINCS+-simple-SHA2 and SPHINCS+-simple-SHAKE." #sphincs-artifact-lengths}
-Parameter ID reference | Parameter name suffix | SPHINCS+ public key | SPHINCS+ secret key | SPHINCS+ signature
-----------------------:| ---------------------:| ------------------- | ------------------- | ------------------
-1                      | 128s                  | 32                  | 64                  | 7856
-2                      | 128f                  | 32                  | 64                  | 17088
-3                      | 192s                  | 48                  | 96                  | 16224
-4                      | 192f                  | 48                  | 96                  | 35664
-5                      | 256s                  | 64                  | 128                 | 29792
-6                      | 256f                  | 64                  | 128                 | 49856
+### Signature Data Digest
 
-### Binding hashes
+SLH-DSA signatures MUST use the associated hash as specified in
+{{tab-slhdsa-hash}} for the digest of signature data according to
+{{I-D.ietf-openpgp-crypto-refresh}} Section 5.2.4. Signatures using other hash
```

Comment:
I just turned the sentence around. I want to say that the signature data digest before the signature algorithm must be done via a hash algorithm from the table. Aron called this "binding hashes".

Reply:
Is "signature data" really the correct term for the data that is signed? To me it seems it could also be understood as the data that represents the signature. Maybe we should better call it "to-be-signed data"?

Reply:
As @fluppe2 pointed out, the above use of the term "signature data" is consistent with the [crypto refresh](https://www.ietf.org/archive/id/draft-ietf-openpgp-crypto-refresh-10.html#section-5.2.4).

Reply:
I suggest to add a clarifying subclause "for the digest of the signature data (data to be signed)"